<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoSysSettingsMiscForm</class>
 <widget class="QWidget" name="GeoSysSettingsMiscForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>701</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>387</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mis. settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="actionsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Actions on visible volumes</string>
     </property>
     <layout class="QGridLayout" name="_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_expand_vols_matname">
        <property name="text">
         <string>Expand</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_expand_vols_matname">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Matching is case-sensitive and wildcards are allowed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Air</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_expand_vols_volname">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Matching is case-insensitive and wildcards are allowed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Vols. with mat:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Vols. named:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_expand_vols_volname">
        <property name="text">
         <string>Expand</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="filtersGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filter volumes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_filter_logVolName">
          <property name="text">
           <string>LogVol's name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_filter_logvolname">
          <property name="toolTip">
           <string>Filter geometry volumes based on the name of the GeoLogVol node. You can use regular expressions and wildcards (*).</string>
          </property>
          <property name="placeholderText">
           <string>ex: Inner.*     [regexp]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_filter_logvolname">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_maxDepth">
          <property name="toolTip">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="statusTip">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="whatsThis">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="text">
           <string>max. depth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_maxDepth">
          <property name="toolTip">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="statusTip">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="whatsThis">
           <string>The maximum number of daughter volumes visited. Note: default is '1';  '0' means 'only root/top volumes';  '-1' means 'MAX', but can be VERY slow depending on the geometry complexity.</string>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>[0: top volumes; -1: All (slower)]</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>How many matching volumes should I show?</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QRadioButton" name="radioButton_StopAtFirst">
           <property name="toolTip">
            <string>Default. Stop after a first match has been found. Useful for a quick check.</string>
           </property>
           <property name="text">
            <string>Show the first match</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_LoopOverAll">
           <property name="toolTip">
            <string>Look for all volumes matching the filter string. ATTENTION, it can take a lof of time.</string>
           </property>
           <property name="text">
            <string>Show all (slower)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Do you want to visit children of matching volumes?</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_VisitAllChildren">
           <property name="toolTip">
            <string>When a volume matches the filter, do visit all its child volumes as well. It can be slow.</string>
           </property>
           <property name="statusTip">
            <string>When a volume matches the filter, do visit all its child volumes as well. It can be slow.</string>
           </property>
           <property name="whatsThis">
            <string>When a volume matches the filter, do visit all its child volumes as well. It can be slow.</string>
           </property>
           <property name="accessibleDescription">
            <string>When a volume matches the filter, do visit all its child volumes as well. It can be slow.</string>
           </property>
           <property name="text">
            <string>Visit All Children</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_DoNotVisitChildren">
           <property name="toolTip">
            <string>Default. When a volume matches the filter, do not visit its child volumes.</string>
           </property>
           <property name="text">
            <string>Do Not Visit Children</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="lockTransp">
           <property name="toolTip">
            <string>When enabling 'Visit All Children', if there are matching daughter to be shown inside a matching mother volume, then the transparency will be automatically set to 50% and set to a given type, to ease the visualization of nested volumes. You can disable this by checking this.</string>
           </property>
           <property name="statusTip">
            <string>When enabling 'Visit All Children', if there are matching daughter to be shown inside a matching mother volume, then the transparency will be automatically set to 50% and set to a given type, to ease the visualization of nested volumes. You can disable this by checking this.</string>
           </property>
           <property name="whatsThis">
            <string>When enabling 'Visit All Children', if there are matching daughter to be shown inside a matching mother volume, then the transparency will be automatically set to 50% and set to a given type, to ease the visualization of nested volumes. You can disable this by checking this.</string>
           </property>
           <property name="accessibleDescription">
            <string>When enabling 'Visit All Children', if there are matching daughter to be shown inside a matching mother volume, then the transparency will be automatically set to 50% and set to a given type. You can disable this by checking this.</string>
           </property>
           <property name="text">
            <string>[lock transparency]</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>(slower)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_filter_reset">
        <property name="toolTip">
         <string>Visualize all volumes by removing all filters.</string>
        </property>
        <property name="text">
         <string>Reset filters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="bottomLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
